lib/Goetas/Mail/ToSwiftMailParser/MimeParser.php

```diff
@@ -123,6 +121,25 @@ protected function parseParts($stream, $partHeaders) {
         }
         return $parts;
     }
+
+    private function getContentType(array $partHeaders)
+    {
+        if (array_key_exists('content-type', $partHeaders)) {
+            return $partHeaders['content-type'];
+        }
+
+        return '';
```

Comment:
Hmm... Interesting. Maybe returning null would be a better idea logically, what do you think?

Yes, it works. It is used in 3 cases:
1. With `stripos`, where it's immediately converted to string: https://github.com/perk11/to-swift-mime-parser/blob/8a8b7eaa321e9925318a925c599b31098f76e37c/lib/Goetas/Mail/ToSwiftMailParser/MimeParser.php#L85-L85 and here https://github.com/perk11/to-swift-mime-parser/blob/8a8b7eaa321e9925318a925c599b31098f76e37c/lib/Goetas/Mail/ToSwiftMailParser/MimeParser.php#L264-L264
2. When constructing exception. It doesn't seem to get used in this case https://github.com/perk11/to-swift-mime-parser/blob/8a8b7eaa321e9925318a925c599b31098f76e37c/lib/Goetas/Mail/ToSwiftMailParser/MimeParser.php#L85-L85
3. In this switch statement, where it doesn't matter: https://github.com/perk11/to-swift-mime-parser/blob/8a8b7eaa321e9925318a925c599b31098f76e37c/lib/Goetas/Mail/ToSwiftMailParser/MimeParser.php#L148-L148